src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -257,7 +258,7 @@ public void validate() throws AppEngineException {
     if (!Files.isRegularFile(JAR_LOCATIONS.get(JAVA_TOOLS_JAR))) {
       throw new AppEngineException(
           "Validation Error: Java Tools jar location '"
-                  + JAR_LOCATIONS.get(JAVA_TOOLS_JAR) + "' is not a file.");
```

Comment:
OK, so I looked it up in the [Google Code Style Guide](https://google.github.io/styleguide/javaguide.html#s4.5.2-line-wrapping-indent), and the correct continuation indent is 4 or more spaces. I think everywhere else in the code, we're using 4 spaces. So, let's stick with that.